threeplay.go

```diff
@@ -36,6 +38,18 @@ type File struct {
 	ErrorDescription     string `json:"error_description"`
 }
 
+type FilesPage struct {
+	Files   []File `json:"files"`
+	Summary `json:"summary"`
+}
+
+type Summary struct {
+	CurrentPage  json.Number `json:"current_page"`
```

Comment:
is there a better for to represent this?
the API returns a number or a string for these fields e.g.:
 `"current_page": 1`
 `"current_page": "1"`